internal/utils/kruize/kruize_api.go

```diff
@@ -45,14 +45,36 @@ func Create_kruize_experiments(experiment_name string, k8s_object []map[string]i
 	}
 	res, err := http.Post(url, "application/json", bytes.NewBuffer(payload))
 	if err != nil {
-		return nil, fmt.Errorf("error Occured while creating experiment: %v", err)
+		log.Errorf("error Occured while creating experiment: %v", err)
 	}
 	defer res.Body.Close()
 	body, _ := io.ReadAll(res.Body)
 	resdata := map[string]interface{}{}
 	if err := json.Unmarshal(body, &resdata); err != nil {
 		return nil, fmt.Errorf("can not unmarshal response data: %v", err)
 	}
+
+	// Temporary fix
+	// Currently, once Kruize pod inits it does not load performance-profile from DB
+	if strings.Contains(resdata["message"].(string), "Performance Profile doesn't exist") {
+		log.Error("Performance profile does not exist")
+		log.Info("Tring to create resource_optimization_openshift performance profile")
+		utils.Setup_kruize_performance_profile()
+	}
+
+	// Re-attempting experiment creation request
+	res, err = http.Post(url, "application/json", bytes.NewBuffer(payload))
+	if err != nil {
+		return nil, fmt.Errorf("error Occured while creating experiment: %v", err)
+	}
```

Comment:
This will duplicate every create experiment request.

Reply:
Can we check the response of create experiment and if it says `Performance Profile doesn't exist` then create the performance profile and give recursive call to `Create_kruize_experiments` function.